src/main.ts

```diff
@@ -369,14 +370,51 @@ async function getServerPath(
 	}
 
 	const workspaceDependency = await getWorkspaceDependency(outputChannel);
+	if (workspaceDependency) {
+		return {
+			bundled: false,
+			workspaceDependency: true,
+			command: workspaceDependency,
+		};
+	}
+
+	if (config.get<boolean | undefined>("biome.searchInPath", true) === true) {
+		outputChannel.appendLine("Searching for Biome in PATH");
+		const biomeInPATH = await findBiomeInPath();
+		if (biomeInPATH) {
+			outputChannel.appendLine(`Biome found in PATH: ${biomeInPATH}`);
+			return {
+				bundled: false,
+				workspaceDependency: false,
+				command: biomeInPATH,
+			};
+		}
+	}
+
+	// Last resort
 	return {
-		bundled: workspaceDependency === undefined,
-		workspaceDependency: workspaceDependency !== undefined,
-		command:
-			workspaceDependency ?? (await getBundledBinary(context, outputChannel)),
+		bundled: true,
+		workspaceDependency: undefined,
+		command: await getBundledBinary(context, outputChannel),
 	};
 }
 
+/**
+ * Attempts top resolve the path to the biome binary from the PATH environment variable.
+ *
+ * This runs `which biome` on Unix-like systems and `where biome` on Windows.
+ */
+async function findBiomeInPath(): Promise<string | undefined> {
+	const which = process.platform === "win32" ? "where" : "which";
```

Comment:
Heads up! The `which` program is not always present in all environments. Perhaps it's better to parse $PATH in-code (it's not that complex) without starting an external process and introducing a new dependency.

![image](https://github.com/biomejs/biome-vscode/assets/10835147/be6536cc-d2be-4ee1-ac70-746c291cd0da)

See https://github.com/biomejs/biome-vscode/issues/51#issuecomment-1890876244

Reply:
Sounds good, I’ll go with your suggestion